testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -744,6 +745,119 @@ struct TenureForkingResult {
     mined_d: MinedNakamotoBlockEvent,
 }
 
+#[test]
+#[ignore]
+/// Test to make sure that the signers are capable of reloading their reward set
+///  if the stacks-node doesn't have it available at the first block of a prepare phase (e.g., if there was no block)
```

Comment:
I think I must be missing something. It looks like it's computing the signer set at the start of the prepare phase. Where is the missed block?

Reply:
Basically, what this test does is it gets the boundary, and then issues 

```
    signer_test.running_nodes.btc_regtest_controller.build_next_block(1);
```

This leads the signers to wake up and try to fetch the reward set, but its not calculated *quite* yet. I think this is kind of a "raced" check, because the node is actually _about_ to process the reward set, and I think would process it if there was just a wait thrown in, but when I run this test against `develop` (i.e., without the patches for the signer), it fails every time for me locally (the signers get an error back from the reward set "fetch", which they treat as meaning that they aren't in the signer set).

Reply:
Okay, got it. Thanks!